### ISSUE TITLE
How to --include-qt-plugins

### ISSUE THREAD

**Issue by JaegerStephan** [Nuitka/Nuitka-Action#15]
Nuitka has the option to include qt plugins with the option `--include-qt-plugins` followed by a list of plugins.
How can this be integrated into the github action because there is no option for qt?

**Comment by kayhayen** [comment]
That is right, and actually it's an extremely import one. Please allow for a bit of time, because I want to use this, to somehow generate this from Nuitka's full set of options, rather than manually aligning them.

**Comment by JaegerStephan** [comment]
Hey @kayhayen!
Sounds good. Van you roughly estimate when it will be available?
Great job, by the way! 🙂

**Comment by kayhayen** [comment]
I estimate that it will not be available before its ready. That will be sooner if you help. Otherwise I am not making promises on time scales outside of Nuitka commercial, and not even there really.

**Comment by JaegerStephan** [comment]
Can you please let me know how I could help. do you have a suggestion what to do next? Is it restricted to Nuitka-Action or are changes for Nuitka necessary as well?

**Comment by kayhayen** [comment]
With the 1.7 release, this will be working, however, for 1.8, there is actually a change, where `sensible` is a default now, and doesn't have to be provided (again) anymore, and to subtract from things, you specify the `--noinclude-qt-plugins` option instead. We need to add that to GitHub actions once 1.8 is released, which however is still a bit of. Obviously only a clone of your PR.
